application/ui/widgets/eh-widgets-dir/eh-widgets/src/helpers/constants.js

```diff
@@ -22,18 +21,35 @@ export const BUTTON_LABELS = {
     DELETE: "Delete",
     EDIT: "Edit",
     CANCEL: "Cancel",
+    REMOVE: "Remove",
 }
 
 // All Modal Labels
 export const MODAL_LABELS = {
     DELETE_BUNDLE_MSG: "Are you sure you want to delete this bundle?",
+    REMOVE_USER_FROM_ORG_MSG: "Are you sure, you want to remove this User from the Organization?",
```

Comment:
Minor, there's an extra comma in this message that shouldn't be there. It should be "Are you sure you want..."

Reply:
Removed the comma